### ISSUE TITLE
notodiff: handling of weights/widths for variable fonts

### ISSUE THREAD

**Issue by marekjez86** [googlei18n/nototools#348]
there are four possible pairings in notodiff
--before <variable> --after <variable>
--before <variable> --after <non-variable> 
--before <non-variable> --after <variable> 
--before <non-variable> --after <non-variable>  

I think <non-variable> font could also be a TTC. How does one pair weights and widths of variable fonts with non-variable one? 

Here's a sample of png obtained from 
--before NotoSansKhmer-Bold --after NotoSansKhmer-GX.ttf 
![uni17a_8](https://cloud.githubusercontent.com/assets/13322623/21268121/402db06a-c362-11e6-93f9-0c8bf2fa1ccf.png)

It appears to me for a variable font notodiff assumes right now a regular weight/width. 

Could it be smarter about it (match what's in the non-variable font) OR allow me to specify?

**Comment by twardoch** [comment]
@marekjez86 It’s a tricky question :) I’ve shared some thoughts here: https://github.com/fonttools/fonttools/pull/763 —

**Comment by jamesgk** [comment]
I don't think this is possible until hb-view gets an option for specifying weight when using a variable font. At that point we'd probably want to allow the user to specify weight for the case when they are comparing two variable fonts.
